tests/load/weaviate/test_pipeline.py

```diff
@@ -67,6 +72,7 @@ def some_data():
     info = pipeline.run(
         some_data(),
     )
+    assert_load_info(info)
 
     data.extend(next(generator_instance2))
     assert_class(info.pipeline, "SomeData", data)
```

Comment:
This probably won't work now since `assert_class` relies on presence of `doc_id` key

Reply:
what `assert_load_info` does is what you do here:
```
package_info = pipeline.get_load_package_info(info.loads_ids[0])
    assert package_info.state == "loaded"
    assert len(package_info.jobs["failed_jobs"]) == 0
    assert len(package_info.jobs["completed_jobs"]) == 1
```

Reply:
Yes, I understand, I was referring this line: 
`assert sorted(objects_without_dlt_keys, key=lambda d: d['doc_id']) == sorted(data, key=lambda d: d['doc_id'])`
It assumes that object will have `doc_id`, but it's only so in one test case.